spec/models/refinery/jobs/job_spec.rb

```diff
@@ -1,10 +1,16 @@
 require 'spec_helper'
+require 'support/helper/generator_helper'
 
 module Refinery
   module Jobs
     describe Job, type: :model do
-      let(:job) {FactoryGirl.create(:job)}
-      describe 'validations' do
+
+      before :each do
+        @job_application = FactoryGirl.create :job_application
+        @job = FactoryGirl.create :job
+      end
+
+      context 'validations' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.